frontend/javascripts/oxalis/model/reducers/skeletontracing_reducer_helpers.ts

```diff
@@ -59,8 +59,13 @@ export function generateTreeName(state: OxalisState, timestamp: number, treeId:
 
   return `${prefix}${Utils.zeroPad(treeId, 3)}`;
 }
+function getMinimumNodeId(trees: TreeMap | MutableTreeMap): number {
+  const minNodeId = _.min(_.flatMap(trees, (tree) => tree.nodes.map((n) => n.id)));
+
+  return minNodeId != null ? minNodeId : Constants.MIN_NODE_ID;
```

Comment:
You're right, the two functions have different behavior in the fallback case. The `getMaximum{Node,Tree}Id` functions return the largest ID encountered. If there are no objects they opt to return the minimal ID - 1 so that the next highest ID would be the minimal ID.

To make `getMinimumNodeId` work uniformly, the fallback case should return Constants.MAX_NODE_ID + 1. However, there is no maximum node id as far as I know :sweat_smile: Since `getMinimumNodeId` is only used for validity checking, returning any number within the valid range is fine, I would argue (as is done currently).